### PR TITLE
Tracking dependencies with dep

### PR DESCRIPTION
**Pull request by arschles**
@markbates I've added the vendor dir to the `.gitignore` since this is a package. Not sure if you'd like me to `dep ensure` in the CI build. Let me know

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+
+[[constraint]]
+  name = "github.com/joho/godotenv"
+  version = "1.2.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/mitchellh/go-homedir"
+
```
Comment (robbyoconnor) [inline Gopkg.toml]: No longer used.

File: Gopkg.lock [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  packages = ["."]
+  revision = "a79fa1e548e2c689c241d10173efd51e5d689d5b"
+  version = "v1.2.0"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/mitchellh/go-homedir"
+  packages = ["."]
+  revision = "b8bc1bf767474819792c23f32d8286a45736f1c6"
+
```
Comment (robbyoconnor) [inline Gopkg.lock]: No longer used.

**Comment by markbates** [comment]
I'm closing this because we've tried it in the past and it caused a lot of problems upstream. Thanks though

**Comment by arschles** [comment]
No worries!

@markbates out of curiosity, I wonder if you could tell me what kind of problems it caused upstream. Might be a good thing to bring up with the dep and/or vgo folks. I'll even volunteer to ferry along some info if you don't have bandwidth for it 😄

**Pull request closed without merging**